lectures/introduction_dynamics/linear_models.md

```diff
@@ -1366,17 +1366,17 @@ Random.seed!(42);
 ```
 
 ```{code-cell} julia
-ϕ1, ϕ2, ϕ3, ϕ4 = 0.5, -0.2, 0, 0.5
-σ = 0.1
-
-A = [ϕ1     ϕ2    ϕ3    ϕ4
-        1.0   0.0   0.0   0.0
-        0.0   1.0   0.0   0.0
-        0.0   0.0   1.0   0.0]
-C = [σ
-        0.0
-        0.0
-        0.0]''
+phi1, phi2, phi3, phi4 = 0.5, -0.2, 0, 0.5
+sigma = 0.1
+
+A = [phi1 phi2 phi3 phi4
+     1.0 0.0 0.0 0.0
+     0.0 1.0 0.0 0.0
+     0.0 0.0 1.0 0.0]
+C = [sigma
+     0.0
+     0.0
+     0.0]''
```

Comment:
This `''` is unclear to me.  See if you can just get rid of it and get the same thing?  Or there are other ways to make this a matrix if it doesn't work.

Reply:
@Jonah-Heyl

Did you look at this?

Reply:
Just now or before?  Because the latest pull this doesn't look changed?